### ISSUE TITLE
GzipDecompressorStream not defined

### ISSUE THREAD

**Issue by jowch** [BioJulia/GenomicAnnotations.jl#8]
Hi, I'm trying to open a `.gbff.gz` file and I'm getting the following error:
```Julia
julia> readgbk(joinpath(DATA_DIR, "genome.gbff.gz"))
ERROR: UndefVarError: GzipDecompressorStream not defined
Stacktrace:
 [1] open(#unused#::Type{GenomicAnnotations.GenBank.Reader}, input::String)
   @ GenomicAnnotations.GenBank ~/.julia/packages/GenomicAnnotations/aGEUx/src/GenBank/reader.jl:31
 [2] readgbk(input::String)
   @ GenomicAnnotations ~/.julia/packages/GenomicAnnotations/aGEUx/src/utils.jl:65
 [3] top-level scope
   @ REPL[5]:1
```
Any help would be much appreciated. Thanks!

**Comment by kdyrhage** [comment]
I've fixed this in v0.3.1 and made it available in General.

**Comment by jowch** [comment]
Thanks! Works now.
